extensions/elytron-security-ldap/runtime/src/main/java/io/quarkus/elytron/security/ldap/config/IdentityMappingConfig.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.elytron.security.ldap.config;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class IdentityMappingConfig {
+
+    /**
+     * The identifier which correlates to the provided user (e.g. "uid")
+     */
+    @ConfigItem
+    public String rdnIdentifier;
```

Comment:
Just to be sure, this is mandatory?

Reply:
It is mandatory, but I can set "uid" to the default value. I think this makes sense.